src/mappings/TinlakeClaimRAD.ts

```diff
@@ -17,7 +17,7 @@ export function handleClaimed(claimed: Claimed): void {
   // add this link to their reward balance and put any
   // claimable rewards into this link, reset linkableRewards to 0
   if (balance.claimable) {
-    link.rewardsAccumulated = balance.linkableRewards
+    link.rewardsAccumulated = link.rewardsAccumulated.plus(balance.linkableRewards)
```

Comment:
Do we know that this is not null here?

Reply:
it's either a BigInt or 0 
`claim.rewardsAccumulated = BigDecimal.fromString('0')`

Reply:
https://github.com/centrifuge/tinlake-subgraph/blob/b3492b645d4050205c6a75779a0f9c770f54d02d/src/domain/RewardLink.ts#L10

Reply:
Perfect, thanks for clarifying!